plugins/infrastructure/skills/memory-keeper/distill.py

```
@@ -0,0 +1,215 @@
+
+def extract_tool_flag_candidates(records: list[dict]) -> list[str]:
+    """
+    Scan notes and fixes_applied for lines that look like tool flags or commands:
+    lines containing ' -', ' --', or backtick-quoted text.
+    """
+    import re
+    flag_pattern = re.compile(r"(`[^`]+`| --?\w[\w\-]*)")
+    seen = set()
+    candidates = []
+    for rec in records:
+        sources = list(rec.get("fixes_applied") or []) + [rec.get("notes") or ""]
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)